packages/uhk-web/src/app/store/reducers/user-configuration.ts

```diff
@@ -528,6 +528,33 @@ export const getMacroMap = (state: State): Map<number, Macro> => {
 };
 export const lastEditedKey = (state: State): LastEditedKey => state.lastEditedKey;
 
+export const getMacroUsage = (state: State): Array<object> => {
```

Comment:
@ert78gb How is it possible that CI has passed, even though indentation uses 2 spaces instead of 4?

Reply:
tslint does not handle correctly the indent size only the type (space/tab).
tslint feature maintenance stopped and everybody has to switch to eslint. eslint handle correctly the indent size.

Reply:
@ert78gb Gotcha, thanks for explaining!
@cokert Please fix the indentation.

Reply:
Fixed, resolving conversation.